docs/pages/develop/development-builds/introduction.mdx

```diff
@@ -1,88 +1,78 @@
 
-Beyond "Hello, world!" and prototypes, you'll quickly encounter limitations and need to build a development build of your app. For example, the Expo Go sandbox environment is limited to only the native packages included in the [Expo SDK](/versions/latest/), while you can include any library in a development build because it's just a normal native app.
+### Remote push notifications
 
-You can think of a development build as your fully customizable version of Expo Go, tailored specifically for your project. Or, conversely, Expo Go is a development environment with a preset runtime.
+While in-app [notifications](https://docs.expo.dev/versions/latest/sdk/notifications/) are available in Expo Go, remote push notifications (that is, sending a push notification from a server into the app) are not. This is because a push notification service should be tied to your own push notifications certificates and while it is possible to make it work in Expo Go, it often caused confusion for production builds. It is therefore better to test remote push notifications in Development Builds so you can ensure parity in behavior between development and production.
 
-</Collapsible>
+### Universal Links / App Links
 
-<Collapsible summary="What other types of builds are there?">
+Both [iOS Universal Links](https://docs.expo.dev/linking/ios-universal-links/) and [Android App Links](https://docs.expo.dev/linking/android-app-links/) require a two-way association between the native app and the website. In particular, it requires for the native app ro include the URL for the lined website. This is not possible with Expo Go due to the aforementioned native code immutability.
 
-- A [production build](/deploy/build-project/) is for the general public, deployed through stores.
-- A [preview build](/build/internal-distribution/) lets your team test your next release, either by installing an APK directly on Android or using ad hoc or enterprise provisioning on iOS.
+### (iOS Device only) open projects using older SDKs
 
-</Collapsible>
+Expo Go can only support one SDK version at a time. So, when we release a new SDK version we need to update Expo Go to support the new SDK version and this will be the only version of Expo Go available to install from the stores.
+
+If you're developing on Android Device, Android Emulator or iOS Simulator, a compatible version of Expo Go can be [downloaded and installed](https://expo.dev/go). The only platform where this is not possible is iPhone devices because apple does not support side-loading older versions of apps.
```

Comment:
```suggestion
If you're developing on an Android Device, Android Emulator, or iOS Simulator, a compatible version of Expo Go can be [downloaded and installed](https://expo.dev/go). The only platform where this is impossible is iPhone devices because Apple does not support side-loading older versions of apps.
```